client-shell/README.md

```diff
@@ -0,0 +1,108 @@
+<!---
+Copyright 2018-2021 Crown Copyright
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--->
+
+# Palisade Shell
+
+The Palisade Shell provides a CLI REPL for access to a cluster and its resources.
+
+> :information_source:
+In the case of the [Palisade examples](https://github.com/gchq/Palisade-examples/), the `context` is limited to just a `purpose` - one of `SALARY`, `DUTY_OF_CARE`, or `STAFF_REPORT`.
```

Comment:
Link points to the GitHub and with it the readme, but there is no explanation of the terms context and purpose. Could benefit from either in the examples readme or here explain what these terms mean.

Reply:
I'll add a reference to the example-library. This should fit together better with the aforementioned better reference to examples